server/container_create.go

```diff
@@ -1038,14 +1038,25 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	}
 
 	// Get imageName and imageRef that are later requested in container status
-	status, err := s.StorageImageServer().ImageStatus(s.ImageContext(), images[0])
-	if err != nil {
-		return nil, err
+	var (
+		imgResult    *storage.ImageResult
+		imgResultErr error
+	)
+	for _, img := range images {
+		status, err := s.StorageImageServer().ImageStatus(s.ImageContext(), img)
+		if err == nil {
+			imgResult = status
```

Comment:
No need for `status` here.  You can just use:

```go
imgResult, err = s.StorageImageServer().ImageStatus(s.ImageContext(), img)
if err == nil {
  break
}
imgResultErr = err
```

Reply:
fixed.